.gitlab/JOBOWNERS

```diff
@@ -165,6 +165,7 @@ pull_test_dockers*                @DataDog/universal-service-monitoring
 
 # Single machine performance
 single_machine_performance*       @DataDog/single-machine-performance
+single-machine-performance*       @DataDog/single-machine-performance
```

Comment:
No, this line is intentional. Some of our existing jobs have names starting with `single_machine_performance` and other existing jobs have names starting with `single-machine-performance`. If there's a strong preference, I can pick one. We use mostly kebab case in `.gitlab/functional_test/regression_detector.yml`, but use snake case in `.gitlab/container_build/docker_linux.yml` and `.gitlab/dev_container_deploy/docker_linux.yml`. The latter two files appear to use snake case for the leading portion of job names as a convention, and I imagine that's why we used snake case there.

Reply:
OK I missed the underscores vs hyphens. I'm not a big fan of mixing things because it's a pitfall I always fall into, but I guess this should be solved elsewhere